From 7135e412c6daa3ed90722f57b6266c1b65bbcb2c Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Wed, 25 Jan 2023 11:00:12 +0100
Subject: [PATCH] struct dwc3: move new members to the end
Patch-mainline: Never (kABI fixup)
References: git-fixes

We have run out of holes for flags, so move them to the end

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/dwc3/core.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/dwc3/core.h b/drivers/usb/dwc3/core.h
index 7247d6f92..f5e3e3ed6 100644
--- a/drivers/usb/dwc3/core.h
+++ b/drivers/usb/dwc3/core.h
@@ -901,11 +901,7 @@ struct dwc3 {
 	struct phy		*usb2_generic_phy;
 	struct phy		*usb3_generic_phy;
 
-	bool			phys_ready;
-
 	struct ulpi		*ulpi;
-	bool			ulpi_ready;
-
 	void __iomem		*regs;
 	size_t			regs_size;
 
@@ -1021,6 +1017,10 @@ struct dwc3 {
 	unsigned		tx_de_emphasis:2;
 
 	u16			imod_interval;
+#ifndef __GENKSYMS__
+	bool			phys_ready;
+	bool			ulpi_ready;
+#endif
 };
 
 #define work_to_dwc(w)		(container_of((w), struct dwc3, drd_work))
-- 
2.39.0

